src/seedu/addressbook/data/person/Address.java

```diff
@@ -40,6 +40,10 @@ public static boolean isValidAddress(String test) {
     public String toString() {
         return value;
     }
+    
+    public String getPrintableString() {
```

Comment:
As discussed in tutorial, you may want to consider the privacy setting when returning a printable string in this system.